build/media_source/com_fields/js/admin-fields-default-batch.es6.js

```diff
@@ -2,18 +2,22 @@
  * @copyright  (C) 2018 Open Source Matters, Inc. <https://www.joomla.org>
  * @license    GNU General Public License version 2 or later; see LICENSE.txt
  */
-document.addEventListener('DOMContentLoaded', () => {
-  'use strict';
+const setUp = (container) => {
+  const batchSelector = container.getElementById('batch-group-id');
+  const batchCopyMove = container.getElementById('batch-copy-move');
 
-  const batchSelector = document.getElementById('batch-group-id');
-  const batchCopyMove = document.getElementById('batch-copy-move');
-  batchCopyMove.classList.add('hidden');
+  if (!container || !batchCopyMove) {
+    return;
+  }
 
+  batchCopyMove.classList.add('hidden');
   batchSelector.addEventListener('change', () => {
     if (batchSelector.value === 'nogroup' || batchSelector.value !== '') {
       batchCopyMove.classList.remove('hidden');
     } else {
       batchCopyMove.classList.add('hidden');
     }
   }, false);
-});
+};
+document.addEventListener('DOMContentLoaded', () => setUp(document));
```

Comment:
If it's `defer` or `type=module` then
```suggestion
setUp(document);
```

Reply:
It does not make diference, but I will allow it here :)